spec/laa_crime_schemas/structs/crime_application_spec.rb

```diff
@@ -22,6 +23,20 @@
           LaaCrimeSchemas::Validator.new(subject.to_json)
         ).to be_valid
       end
+
+      it 'has an undetermined offence class' do
+        expect(subject.case_details.offence_class).to eq(nil)
+      end
+    end
+
+    context 'for an valid crime application object with multi class offences' do
+      let(:attributes) do
+        JSON.parse(file_fixture(multi_class_offence_fixture).read)
```

Comment:
I'm a bit worried about the proliferation of fixtures in this gem. An alternative approach might be to include the trait within a mock object and use that for the spec. Perhaps even move the offence_class specs so they are tested independent of the CrimeApplication object.